common/src/sql/dbinit.sql

```diff
@@ -81,6 +81,11 @@ CREATE TABLE omicron.public.sled (
     /* Idenfities if this Sled is a Scrimlet */
     is_scrimlet BOOL NOT NULL,
 
+    /* Baseboard information about the sled */
+    baseboard_identifier STRING(63) NOT NULL,
+    baseboard_model STRING(63) NOT NULL,
+    baseboard_revision INT8 NOT NULL,
```

Comment:
Does baseboard carry some semantic meaning that customers would understand? I'd based the context of the names off of [RFD-308#3.3](https://rfd.shared.oxide.computer/rfd/308#_human_readable_text) in #2231.

Reply:
Digging around and it seems like this is fairly standard:
- windows: `wmic baseboard`
- linux: `sudo dmidecode -t baseboard`

Reply:
We may want to call it something else in the customer visible view (e.g., Gimlet ID or Compute Sled ID or whatever marketing name we give the sled), but I think it's good that we're calling it one thing throughout the internals of the stack.